engine/api/workflow/dao_node_run_job.go

```diff
@@ -141,8 +141,8 @@ func DeleteNodeJobRuns(db gorp.SqlExecutor, nodeID int64) error {
 	return err
 }
 
-//UpdateNodeJobRun updates a workflow_node_run_job
-func UpdateNodeJobRun(db gorp.SqlExecutor, store cache.Store, p *sdk.Project, j *sdk.WorkflowNodeJobRun) error {
+//UpdateNodeJobRunAndExecute updates a workflow_node_run_job then run workflow.execute
```

Comment:
It's not the good way to fix it.
I already have a fix for that on a dev branch. I will make the PR today (i hope)